<!DOCTYPE html>
<html style="font-size: 16px;" lang="en"><head>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta charset="utf-8">
    <meta name="keywords" content="Brandon M. Lê">
    <meta name="description" content="">
    <title>Research</title>
    <link rel="stylesheet" href="nicepage.css" media="screen">
<link rel="stylesheet" href="Research.css" media="screen">
    <script class="u-script" type="text/javascript" src="jquery.js" defer=""></script>
    <script class="u-script" type="text/javascript" src="nicepage.js" defer=""></script>
    <meta name="generator" content="Nicepage 4.19.3, nicepage.com">
    
    <link id="u-theme-google-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i|Open+Sans:300,300i,400,400i,500,500i,600,600i,700,700i,800,800i">
    
    
    
    
    
    
    
    <script type="application/ld+json">{
		"@context": "http://schema.org",
		"@type": "Organization",
		"name": "Site1"
}</script>
    <meta name="theme-color" content="#478ac9">
    <meta property="og:title" content="Research">
    <meta property="og:type" content="website">
  </head>
  <body class="u-body u-overlap u-overlap-contrast u-xl-mode" data-lang="en"><header class="u-clearfix u-header" id="sec-067f" data-animation-name="" data-animation-duration="0" data-animation-delay="0" data-animation-direction=""><div class="u-clearfix u-sheet u-valign-middle u-sheet-1">
        <nav class="u-align-right u-menu u-menu-one-level u-offcanvas u-menu-1">
          <div class="menu-collapse" style="font-size: 1rem; letter-spacing: 0px;">
            <a class="u-button-style u-custom-left-right-menu-spacing u-custom-padding-bottom u-custom-text-active-color u-custom-top-bottom-menu-spacing u-nav-link u-text-active-palette-1-base u-text-hover-palette-2-base" href="#">
              <svg class="u-svg-link" viewBox="0 0 24 24"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#menu-hamburger"></use></svg>
              <svg class="u-svg-content" version="1.1" id="menu-hamburger" viewBox="0 0 16 16" x="0px" y="0px" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns="http://www.w3.org/2000/svg"><g><rect y="1" width="16" height="2"></rect><rect y="7" width="16" height="2"></rect><rect y="13" width="16" height="2"></rect>
</g></svg>
            </a>
          </div>
          <div class="u-custom-menu u-nav-container">
            <ul class="u-nav u-spacing-10 u-unstyled u-nav-1"><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-white u-text-body-alt-color u-text-hover-palette-2-base" href="Home.html" style="padding: 10px;">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-white u-text-body-alt-color u-text-hover-palette-2-base" href="CV.html" style="padding: 10px;">CV</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-white u-text-body-alt-color u-text-hover-palette-2-base" href="Research.html" style="padding: 10px;">Research</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-white u-text-body-alt-color u-text-hover-palette-2-base" href="Teaching.html" style="padding: 10px;">Teaching</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-white u-text-body-alt-color u-text-hover-palette-2-base" href="Service-and-Outreach.html" style="padding: 10px;">Service and Outreach</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-white u-text-body-alt-color u-text-hover-palette-2-base" href="Communication.html" style="padding: 10px;">Communication</a>
</li></ul>
          </div>
          <div class="u-custom-menu u-nav-container-collapse">
            <div class="u-black u-container-style u-inner-container-layout u-opacity u-opacity-95 u-sidenav">
              <div class="u-inner-container-layout u-sidenav-overflow">
                <div class="u-menu-close"></div>
                <ul class="u-align-center u-nav u-popupmenu-items u-unstyled u-nav-2"><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Home.html">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="CV.html">CV</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Research.html">Research</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Teaching.html">Teaching</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Service-and-Outreach.html">Service and Outreach</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Communication.html">Communication</a>
</li></ul>
              </div>
            </div>
            <div class="u-black u-menu-overlay u-opacity u-opacity-70"></div>
          </div>
        </nav>
      </div></header> 
    <section class="skrollable u-align-left u-clearfix u-image u-shading u-section-1" src="" data-image-width="3542" data-image-height="2359" id="sec-5754">
      <div class="u-clearfix u-sheet u-valign-middle u-sheet-1">
        <h1 class="u-custom-font u-text u-text-default u-title u-text-1">Brandon M. Lê</h1>
        <p class="u-large-text u-text u-text-variant u-text-2">PhD Candidate in Genetics &amp; Genomics, Duke University</p>
      </div>
    </section>
    <section class="u-align-center u-clearfix u-section-2" id="sec-8c89">
      <div class="u-clearfix u-sheet u-sheet-1">
        <h2 class="u-text u-text-default u-text-1">Research</h2>
      </div>
    </section>
    <section class="u-align-center u-clearfix u-grey-5 u-section-3" id="sec-7f0c">
      <div class="u-clearfix u-sheet u-valign-middle u-sheet-1">
        <p class="u-align-left u-text u-text-1"> My research interests focus on using novel computational techniques to enhance prediction of genotype-to-phenotype associations. My academic and research backgrounds are rooted in both biology and computer science, allowing me to integrate concepts and methods from both fields in my research. As an undergraduate at Brown University, I conducted research under Dr. Irina Arkhipova at the Marine Biological Laboratory (MBL). My project investigated the genome of a novel parasitic wasp whose body size is greatly reduced compared to related wasps. The findings from this study resulted in a co-authorship publication, and I presented these findings at the Mobile Genetic Elements conference hosted at the MBL. As a current graduate student at Duke University, I work with Dr. Allison Ashley-Koch on the genetic modifiers of sickle cell disease (SCD). Our lab is broadly interested in the genetic epidemiology of human genetic disorders, and my project and program will comprehensively prepare me for a career in academia. My project aims to impute multi-omics profiles in SCD patients, which will then enable prediction of renal outcomes in those patients, utilizing novel machine learning methods to strengthen the power of the imputation algorithms. My program and institution provide multiple opportunities for professional development, career exploration, skills development, and mentorship opportunities, in which I heavily engage. These opportunities also support my own personal goals of becoming a well-rounded scientist, and making science accessible to both the public and to future scientists. I am a first-generation college student: my family has always emphasized the value of education, and I want to help others in similar positions who want to pursue higher education. Overall, my goal is to become a scientist proficient in both research and mentorship, able to pose and investigate scientific inquiries and translate their results for the greater community to benefit. <br>
          <br>As an undergraduate student, I conducted research at the Marine Biological Laboratory over two summers. Under the guidance of Dr. Irina Arkhipova, I investigated the genome of a novel species of parasitic wasp (Megaphragma amalphitanum), focusing on how transposable elements (TEs) may have affected its evolution. Wasps in Megaphragma have microscopic body sizes, and our hypothesis was that TEs affected genome composition and contributed to body size reduction. I performed de novo detection and annotation of transposable elements in M. amalphitanum, writing and maintaining various programs to determine the wasp’s TE composition and history. Compared to related wasps, M. amalphitanum had similar TE composition but different TE activity, where it is currently experiencing less TE integration compared to its evolutionary past. This could be attributed to recently-acquired genome defense machinery that prevents rampant TE accumulation in its genome. <br>
          <br>As a graduate student, I conduct research on the genetic modifiers of sickle cell disease (SCD). SCD pathogenesis varies widely between individual patients despite the constancy of a beta globin point mutation, suggesting that other factors influence SCD disease progression. In support of this research, our lab has collected data on SCD nephropathy (SCDN) and multi-omics profiles, which includes genomes, metabolomics, and proteomics. This data is organized in a well-structured SCD patient cohort as part of the NHLBI TOPMed program. In conjunction with several other TOPMed SCD cohorts, our aim is to better characterize omic factors contributing to SCDN progression. The omics data available for analysis are not always complete, and there exists potential gains from imputation of currently available data to boost the coverage and statistical power of these analyses. My project will develop deep learning methodologies to impute missing multi-omics data and predict SCDN outcomes given a patient’s existing set of multi-omics profiles. GWASes and meta-analyses of the TOPMed SCD cohorts have been performed, and the proposed work will utilize these results to construct, train, and test neural networks capable of both imputation and omics-phenotype association.<br>
          <br>A select list of projects and publications are displayed below.
        </p>
      </div>
    </section>
    <section class="u-clearfix u-section-4" id="sec-035e">
      <div class="u-clearfix u-sheet u-valign-middle u-sheet-1">
        <div class="u-clearfix u-expanded-width u-gutter-10 u-layout-wrap u-layout-wrap-1">
          <div class="u-gutter-0 u-layout">
            <div class="u-layout-col">
              <div class="u-size-30">
                <div class="u-layout-row">
                  <div class="u-align-left u-container-style u-layout-cell u-shape-rectangle u-size-60 u-layout-cell-1">
                    <div class="u-container-layout u-container-layout-1">
                      <p class="u-text u-text-1">Manuscript in prep., 2022</p>
                    </div>
                  </div>
                </div>
              </div>
              <div class="u-size-30">
                <div class="u-layout-row">
                  <div class="u-container-style u-layout-cell u-size-28 u-layout-cell-2">
                    <div class="u-container-layout u-container-layout-2">
                      <img class="u-expanded-width u-image u-image-contain u-image-default u-preserve-proportions u-image-1" src="images/OMG-SCD_minDP10_MAF5_eGFR90_manhattan_plot.png" alt="" data-image-width="480" data-image-height="480">
                    </div>
                  </div>
                  <div class="u-container-style u-layout-cell u-size-32 u-layout-cell-3">
                    <div class="u-container-layout u-container-layout-3">
                      <p class="u-text u-text-2"> Sickle cell disease (SCD) is a blood disorder that causes sickling of red blood cells (RBCs), hemolysis, and damage to multiple organ systems. Renal dysfunction in adults with SCD is one such damage that may occur and is highly associated with early mortality. However, not all patients develop significant renal dysfunction, suggesting that factors beyond the primary beta globin mutation impact risk. Sickle cell disease nephropathy (SCDN), defined by the presence of proteinuria and low estimated glomerular filtration rate (eGFR), is strongly associated with mortality. Previous studies have implicated genetic risk factors for SCDN, and our group has preliminary data suggesting certain plasma metabolites and proteins are associated with SCDN phenotypes.<br>
                        <br>Using data from the NHLBI TOPMed SCD cohorts, genome-wide association studies (GWASes) were performed on each cohort to determine if any common single-nucleotide polymorphisms are statistically associated with SCDN outcomes. Meta-analyses integrating the summary statistics from the above GWASes were also performed, to identify variants common to multiple cohorts. This work was performed on the NHLBI BioData Catalyst ecosystem, powered by the Seven Bridges platform. The GWAS pipelines developed in this project will inform future studies by illuminating important SCD pathophysiology and facilitate future genomics analyses. 
                      </p>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <h3 class="u-text u-text-3"> Genome-wide association studies of renal outcomes in the TOPMed sickle cell disease cohorts</h3>
      </div>
    </section>
    <section class="u-clearfix u-grey-5 u-section-5" id="carousel_9f5c">
      <div class="u-clearfix u-sheet u-valign-middle u-sheet-1">
        <div class="u-clearfix u-expanded-width u-gutter-10 u-layout-wrap u-layout-wrap-1">
          <div class="u-gutter-0 u-layout">
            <div class="u-layout-col">
              <div class="u-size-30">
                <div class="u-layout-row">
                  <div class="u-align-left u-container-style u-layout-cell u-shape-rectangle u-size-60 u-layout-cell-1">
                    <div class="u-container-layout u-container-layout-1">
                      <h3 class="u-text u-text-1"> Modeling sickle cell disease phenotypes through supervised learning of patient multi-omic data</h3>
                      <p class="u-text u-text-2">Project in progress, 2022</p>
                    </div>
                  </div>
                </div>
              </div>
              <div class="u-size-30">
                <div class="u-layout-row">
                  <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-2">
                    <div class="u-container-layout u-container-layout-2">
                      <img class="u-align-center u-image u-image-default u-image-1" src="images/g2655.png" alt="" data-image-width="4196" data-image-height="1058">
                      <p class="u-text u-text-3"> Sickle cell disease (SCD) is a blood disorder that causes sickling of red blood cells (RBCs), hemolysis, and damage to multiple organ systems. Renal dysfunction in adults with SCD is one such damage that may occur and is highly associated with early mortality. However, not all patients develop significant renal dysfunction, suggesting that factors beyond the primary beta globin mutation impact risk. Sickle cell disease nephropathy (SCDN), defined by the presence of proteinuria and low estimated glomerular filtration rate (eGFR), is strongly associated with mortality. Previous studies have implicated genetic risk factors for SCDN, and our group has preliminary data suggesting certain plasma metabolites and proteins are associated with SCDN phenotypes. However, missing omics data reduce the power to make these discoveries. This proposal aims to model the correlations between individual omics datasets to impute missing omics data and discover associations with proteinuria and eGFR outcomes in several TOPMed SCD patient cohorts: OMG-SCD, Walk-PHaSST, PUSH, and REDS III Brazil. All datasets contain wholegenome sequencing data, and a subset of these datasets contains metabolomic and proteomic data.<br>
                        <br>Using machine learning methods, this proposal will construct several deep learning models for the purpose of 1) imputing missing data between omics datasets, and 2) predicting omic variants associated with SCDN. Aim 1 will develop a neural network model whose function is to impute missing proteomic and metabolomic data within patients’ omics profiles, given currently-available omics data from the SCD cohorts and the model’s prior training. Prior studies have put forward statistically significant genomic and omic variants associated with SCDN outcomes, which will serve as training and validation datasets. Aim 1 will supplement Aim 2 by providing additional data from which to make associations between biomarkers and SCDN outcomes. Aim 2 will develop a separate deep learning network that directly investigates the genomic and omic variants associated with SCDN in the more complete data set derived from Aim 1. Given patients’ SCDN phenotypes and genetic/omic datasets, this model will perform association testing to discover significant associations. The deep learning model derived from aim 2 will provide a unified framework to simultaneously analyze multiple omics datasets and provide more comprehensive coverage of associations between biomarkers and outcomes. The predictive models developed in this proposal will inform future studies through combining machine learning tools with multi-omics data, thereby illuminating important SCD pathophysiology and demonstrating the utility of machine learning to facilitate multi-omics analyses.<br>
                      </p>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-6" id="carousel_95a3">
      <div class="u-clearfix u-sheet u-valign-middle u-sheet-1">
        <div class="u-clearfix u-expanded-width u-gutter-10 u-layout-wrap u-layout-wrap-1">
          <div class="u-gutter-0 u-layout">
            <div class="u-layout-col">
              <div class="u-size-30">
                <div class="u-layout-row">
                  <div class="u-align-left u-container-style u-layout-cell u-shape-rectangle u-size-60 u-layout-cell-1">
                    <div class="u-container-layout u-container-layout-1">
                      <h3 class="u-text u-text-1"> Genomic signatures of miniaturization in the parasitoid wasp <span style="font-style: italic;">Megaphragma amalphitanum</span>
                      </h3>
                      <p class="u-text u-text-2">Published as doi:<a href="https://doi.org/10.1371%2Fjournal.pone.0226485" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6927652&amp;issue-id=346951&amp;journal-id=440&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">10.1371/journal.pone.0226485</a>, 2019
                      </p>
                    </div>
                  </div>
                </div>
              </div>
              <div class="u-size-30">
                <div class="u-layout-row">
                  <div class="u-align-center u-container-style u-layout-cell u-size-60 u-layout-cell-2">
                    <div class="u-container-layout u-valign-bottom u-container-layout-2">
                      <img class="u-align-center u-image u-image-default u-image-1" src="images/Megaphragmaamalphitanum.png" alt="" data-image-width="1000" data-image-height="600">
                      <p class="u-align-left u-text u-text-3"> Body size reduction, also known as miniaturization, is an important evolutionary process that affects a number of physiological and phenotypic traits and helps animals conquer new ecological niches. However, this process is poorly understood at the molecular level. Here, we report genomic and transcriptomic features of arguably the smallest known insect–the parasitoid wasp,&nbsp;Megaphragma amalphitanum&nbsp;(Hymenoptera: Trichogrammatidae). In contrast to expectations, we find that the genome and transcriptome sizes of this parasitoid wasp are comparable to other members of the Chalcidoidea superfamily. Moreover, compared to other chalcid wasps the gene content of&nbsp;M.&nbsp;amalphitanum&nbsp;is remarkably conserved. Intriguingly, we observed significant changes in&nbsp;M.&nbsp;amalphitanum&nbsp;transposable element dynamics over time, in which an initial burst was followed by suppression of activity, possibly due to a recent reinforcement of the genome defense machinery. Overall, while the&nbsp;M.&nbsp;amalphitanum&nbsp;genomic data reveal certain features that may be linked to the unusual biological properties of this organism, miniaturization is not associated with a large decrease in genome complexity.</p>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <style class="u-overlap-style">.u-overlap:not(.u-sticky-scroll) .u-header {
background-color: rgba(0,51,102,0.64) !important
}</style>
    
    
    <footer class="u-align-center u-clearfix u-footer u-grey-80 u-footer" id="sec-3e89"><div class="u-clearfix u-sheet u-valign-middle u-sheet-1">
        <p class="u-small-text u-text u-text-variant u-text-1"> © 2022 Brandon M. Lê</p>
      </div></footer>
    <section class="u-backlink u-clearfix u-grey-80">
      <a class="u-link" href="https://nicepage.com/website-templates" target="_blank">
        <span>Website Templates</span>
      </a>
      <p class="u-text">
        <span>created with</span>
      </p>
      <a class="u-link" href="" target="_blank">
        <span>Website Builder Software</span>
      </a>. 
    </section>
  
</body></html>